<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-807AG9O" failures="0" tests="1" name="Test1" time="62.703" errors="0" timestamp="2021-09-25T07:55:53 IST">
  <testcase classname="testcases.TC_01_AddEmployee" name="@AfterClass tearDown" time="0.181">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Employee details do not exist expected [mjDhvP LYI ejfc] but found [Odis Adalwin]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Employee details do not exist expected [mjDhvP LYI ejfc] but found [Odis Adalwin]
at testcases.TC_01_AddEmployee.tearDown(TC_01_AddEmployee.java:305)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase classname="testcases.TC_01_AddEmployee" name="add_Employee" time="60.201"/>
</testsuite> <!-- Test1 -->
